rp2040-hal/src/rom_data.rs

```diff
@@ -150,6 +150,11 @@ macro_rules! declare_rom_function {
         }
 
         $(#[$outer])*
+        /// # Safety
+        ///
+        /// This is a low-level C function. It may be difficult to call safely from
+        /// Rust. If in doubt, check the RP2040 datasheet for details and do your own
+        /// safety evaluation.
```

Comment:
```suggestion
        /// # Safety
        ///
        /// This is a low-level C function. It may be difficult to call safely from
        /// Rust. If in doubt, check the RP2040 datasheet for details and do your own
        /// safety evaluation.
        ///
        /// For more informations on the considerations when calling C functions from
        /// Rust, see [the FFI chapter in the Secure Rust Guidelines book] as well as
        /// the section on [FFI in the Nomicon].
        ///
        /// [the FFI chapter in the Secure Rust Guidelines book]: https://anssi-fr.github.io/rust-guide/07_ffi.html
        /// [FFI in the Nomicon]: https://doc.rust-lang.org/nomicon/ffi.html
```